src/algorithmForNN/pointCloudUtils.py

```diff
@@ -151,6 +151,111 @@ def get_area_of_plane(points: o3d.geometry.PointCloud, plane_model: list) -> flo
     return current_list[1] * current_list[2]
 
 
+def get_calibration_matrix_from_calib_file(path_to_calibration_file: str) -> np.ndarray:
+    calib_matrix = np.zeros((4, 4))
+    with open(path_to_calibration_file) as file:
+        (
+            calib_matrix[0, 0],
+            calib_matrix[0, 1],
+            calib_matrix[0, 2],
+            calib_matrix[0, 3],
+            calib_matrix[1, 0],
+            calib_matrix[1, 1],
+            calib_matrix[1, 2],
+            calib_matrix[1, 3],
+            calib_matrix[2, 0],
+            calib_matrix[2, 1],
+            calib_matrix[2, 2],
+            calib_matrix[2, 3],
+        ) = list(map(float, file.readlines()[4][4:].rstrip().split(" ")))
+        calib_matrix[3, 3] = 1
+
+    return calib_matrix
+
+
+def get_position_matrix_from_poses_file(
+    path_to_poses_file: str, frame_number: int
+) -> np.ndarray:
+    pose_matrix = np.zeros((4, 4))
```

Comment:
Same?

Reply:
Better to read poses all together one time opening file, not to access storage every time. They are quite light.